api.raml

```diff
@@ -928,3 +977,119 @@ schemas:
+/method:
+  post:
+    description: Create a new recovery method and send verification email.
+    body:
+      application/json:
+        example: |
+          {
+            "employee_id": "12345",
+            "value": "987654"
+          }
+    responses:
+      200:
+        description: Recovery method created and email sent.
+        body:
+          application/json:
+            schema: MethodResponse
+            example: |
+              {
+                "uid": "12345678901234567890123456789012",
+                "value": "email@example.org"
+              }
+      400:
+        description: Missing parameter
+      409:
+        description: New method matches an already existing verified method
+      500:
+        description: Database or other system error
+  /{uid}:
+    get:
+      description: Retrieve a specific recovery method.
+      body:
+        application/json:
+          example: |
+            {
+              "employee_id": "12345"
+            }
+      responses:
+        200:
+          description: Recovery method found
+          body:
+            application/json:
+              schema: MethodResponse
+              example: |
+                {
+                  "uid": "12345678901234567890123456789012",
+                  "value": "email@example.org"
+                }
+        404:
+          description: User not found or method not found
+    delete:
+      description: Delete a specific recovery method.
+      body:
+        application/json:
+          example: |
+            {
+              "employee_id": "12345"
+            }
+      responses:
+        200:
+          description: Recovery method found
+          body:
+            application/json:
+              schema: MethodResponse
+              example: |
+                {
+                  "uid": "12345678901234567890123456789012",
+                  "value": "email@example.org"
+                }
+        404:
+          description: User not found or method not found
+        500:
+          description: Delete failed
+    /verify:
+      put:
+        description: Verify the recovery method by a matching code.
+        body:
+          application/json:
+            example: |
+              {
+                "employee_id": "12345",
+                "code": "987654"
+              }
+        responses:
+          200:
+            description: Provided code matches original code
+            body:
+              application/json:
+                schema: MethodResponse
+                example: |
+                  {
+                    "uid": "12345678901234567890123456789012",
+                    "value": "email@example.org"
+                  }
+          400:
+            description: Missing or invalid code
+          404:
+            description: User not found or method not found
+          500:
+            description: Database or other system error
+    /resend:
+      put:
+        description: Resend the recovery email
```

Comment:
This could also be an SMS (not just an email), right? If so, that might be worth including in the docs here:
`description: Resend the recovery email/text` or `description: Resend the recovery message`

Reply:
As of today, recovery methods are always email addresses. I'll make that note in the RAML file.